jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/JibCli.java

```diff
@@ -83,7 +85,8 @@
       required = true,
       paramLabel = "<target-image>",
       description =
-          "The destination image reference or jib style url,%nexamples:%n gcr.io/project/image,%n registry://image-ref,%n docker://image,%n tar://path")
+          "The destination image reference or jib style url,%nexamples:%n gcr.io/project/image,%n registry://image-ref,%n docker://image,%n tar://path",
+      scope = CommandLine.ScopeType.INHERIT)
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private String targetImage;
 
```

Comment:
Should we also move `context`, `build-file` and `parameter` to Build.java?

Reply:
Good question. By `parameters`, you mean the three hidden parameters? I think they should be shared. And for `context` and `build-file`, it makes sense to move `Build.java`, since we are not using it in `Jar.java` at the moment.

Reply:
Sounds good! Oh sorry by parameter, I meant the `templating parameter`.

Reply:
I don't think I'll come through all the options. Just make sure to move options only specific to the `build` command (like those three) to `Build.java`.

BTW, the doc says that `INHERIT` cannot be used in `@ArgGroup`, which we have in some places. But I guess it's irrelevant to us?

Reply:
Ah good point. Applying the `INHERIT` feature for the `@ArgGroup`  still seems to be a work in progress. Currently, using `INHERIT` on the options in the arg group allows those options to be inherited by all the subcommands but it doesn't let the other arg group attributes to be inherited (for example, validation for exclusivity). This might be something that we need when we are handling credentials for the `jar` and `build` subcommands. Hm it is probably better to use Mixins for now as support for `INHERIT` is still a little new.